src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -481,9 +484,9 @@ public void deleteEquipmentIndexes(UUID networkUuid) {
     }
 
     public CreatedStudyBasicInfos insertStudy(UUID studyUuid, String userId, NetworkInfos networkInfos,
-            String caseFormat, UUID caseUuid, boolean casePrivate, String caseName, LoadFlowParametersEntity loadFlowParameters, UUID importReportUuid) {
+                                              String caseFormat, UUID caseUuid, boolean casePrivate, String caseName, LoadFlowParametersEntity loadFlowParameters, ShortCircuitParametersEntity shortCircuitParametersEntity, UUID importReportUuid) {
```

Comment:
You realigned the line. I don't know if it was intented.

Reply:
fixed